ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__nightly-4.11-upgrade-from-stable-4.10.yaml

```diff
@@ -69,6 +69,19 @@ tests:
     pre:
     - chain: cucushift-installer-rehearse-azure-ipi-provision
     workflow: cucushift-upgrade
+- as: azure-ipi-disconnected
+  interval: 24h
```

Comment:
Running every profile with 24h as interval is too much as we are adding more and more profiles.

Please rebase the PR and pick up a proper one.

Reply:
Thanks for the suggestion.